docs/MS Excel/Формулы/функции суммы, среднего и количества.md

```diff
@@ -0,0 +1,52 @@
+# **Функции суммы, среднего и количества**
+
+## **Функция СУММ**
+
+Это одна из математических функций.
+ Она нужна, чтобы автоматически посчитать сумму чисел в выбранном диапазоне.
+ ### **Как работать с функцией**
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)